src/tracing/node_trace_writer.h

```diff
@@ -61,7 +61,7 @@ class NodeTraceWriter : public AsyncTraceWriter {
   int highest_request_id_completed_ = 0;
   int total_traces_ = 0;
   int file_num_ = 0;
-  const std::string& log_file_pattern_;
+  std::string log_file_pattern_;
```

Comment:
👍 
I've also hit that bug in https://github.com/nodejs/node/commit/6c24c3b8db54d29e9191899b302b3cd486afbd86